nemo/nemo/utils/exp_logging.py

```diff
@@ -144,8 +144,9 @@ def get_tb_writer(self, tb_dir=None, exist_ok=True):
                 self.tb_writer = SummaryWriter(self.tb_dir)
             except ImportError:
                 self.tb_writer = None
-                self.logger.info('Not using TensorBoard.')
-                self.logger.info('Install tensorboardX to use TensorBoard')
+                # giving NoneType Error, so commenting out temporarily
```

Comment:
please add check "if self.logger is not None" to guard it against NoneType error